Lecture5/index.js

```diff
@@ -0,0 +1,21 @@
+function debounce(func, delay) {
+    var timeoutID;
+
+    function debounced(value) {
+
+        if (timeoutID !== undefined) clearTimeout(timeoutID);
```

Comment:
Можно укоротить проверку

```suggestion
        if (timeoutID) clearTimeout(timeoutID);
```